Makefile

```
@@ -17,72 +17,57 @@
 #
 PKGNAME=dasbus
 VERSION=$(shell awk '/Version:/ { print $$2 }' python-$(PKGNAME).spec)
-
-PREFIX=/usr
-
+TAG=v$(VERSION)
 PYTHON?=python3
 COVERAGE?=coverage3
 
-build:
-	$(PYTHON) setup.py build
-
+.PHONY: clean
 clean:
-	$(PYTHON) setup.py -q clean --all
+	git clean -fdx
+
+.PHONY: check
+check:
+	@echo "*** Running pylint ***"
+	$(PYTHON) -m pylint dasbus/ tests/
 
+.PHONY: coverage
 coverage:
 	@echo "*** Running unittests with $(COVERAGE) ***"
 	PYTHONPATH=. $(COVERAGE) run --branch -m unittest discover -v -s tests/
 	$(COVERAGE) report -m --include="dasbus/*" | tee coverage-report.log
 
-check:
-	@echo "*** Running pylint ***"
-	$(PYTHON) -m pylint dasbus/ tests/
+.PHONY: changelog
+changelog:
+	@git log --pretty="format:- %s (%ae)" $(TAG).. | sed -e 's/@.*)/)/' | grep -v "Merge pull request"
```

```suggestion
	@git log --no-merges --pretty="format:- %s (%ae)" $(TAG).. | sed -e 's/@.*)/)/'
```

Seems like a better solution to me.